pkgs/by-name/im/imgpkg/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+let
+  pname = "imgpkg";
+  version = "0.42.1";
+
+  src = fetchFromGitHub {
+    owner = "carvel-dev";
+    repo = "imgpkg";
+    rev = "v${version}";
+    hash = "sha256-n8v27FpFpx0PBu/cCHgq128+fAjSGTEdLCG/g7MHCVg";
+  };
+in
+buildGoModule {
+  inherit pname version src;
+
+  vendorHash = null;
+
+  subPackages = [ "cmd/imgpkg" ];
+
+  CGO_ENABLED = "0";
```

Comment:
```suggestion
  env.CGO_ENABLED = "0";
```

Reply:
@BenChand: whoops, that was my bad. I assumed it was an environment variable to be passed to the builder, but it looks like the Go framework handles it differently. Remove the `env.`.

Reply:
Ah no worries, will revert